bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -35,7 +35,14 @@ limitations under the License.
             <dependency>
                 <groupId>com.google.cloud</groupId>
                 <artifactId>google-cloud-bom</artifactId>
-                <version>${google-cloud-bom.version}</version>
+                <version>${google-cloud.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
google-cloud-clients already imports google-cloud-bom. Does this not get propagated?

Reply:
Interesting.  Apparently it is propagated.  I'll remove it.